#! /bin/sh /usr/share/dpatch/dpatch-run
## 03_diffoptstypo.dpatch by  <crafterm@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Diff opts typo fix

@DPATCH@

--- cvsps-2.1-orig/cvsps.1	2005-05-26 05:39:40.000000000 +0200
+++ cvsps-2.1/cvsps.1	2005-07-28 15:17:48.885112048 +0200
@@ -83,7 +83,7 @@
 disable the use of rlog internally.  Note: rlog is
 required for stable PatchSet numbering.  Use with care.
 .TP
-.B \-\-diffs\-opts <option string>
+.B \-\-diff\-opts <option string>
 send a custom set of options to diff, for example to increase
 the number of context lines, or change the diff format.
 .TP
